OPENEUROPA-722: Make sure the default body classes are output

pfrenssen opened the pull request:
The theme misses the important body classes that are used to identify whether the user is logged in, visiting an admin page, the node type etc. Without these it will not be possible to write effective JS, and the functional tests are also failing.

Ref. `template_preprocess_html()`
Ref. `html.html.twig` in the Classy theme.

brummbar commented on templates/html/html.html.twig:
```diff
@@ -14,7 +22,7 @@
     <js-placeholder token="{{ placeholder_token }}">
   </head>
-  <body{{ attributes.addClass('language-' ~ html_attributes.lang).addClass('ecl-typography') }}>
+  <body{{ attributes.addClass('language-' ~ html_attributes.lang).addClass('ecl-typography').addClass(body_classes) }}>
```
We could merge these two classes added here to the newly created `body_classes` array.

pfrenssen commented:
Thanks @brummbar, great suggestion!

I've added the test and rebased the PR so that the test runs first. It nicely demonstrates that the body classes are not present, since in order to detect if a user is logged in, it verifies the body classes first, and then resorts to finding a logout link. It can't do that:

```
--- Failed steps:
001 Scenario:                                                      # tests/features/authentication.feature:7
      Given I am logged in as a user with the "authenticated" role # tests/features/authentication.feature:10
        Unable to determine if logged in because 'log_out' link cannot be found for user 'fBiZSXqm' with role 'authenticated' (Exception)
```

Medi4tor commented on composer.json:
```diff
@@ -57,6 +57,14 @@
                 "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch"
             }
         },
+        "patches-ignore": {
```
Hi,
Why do we have this ignore section in this commit? 
Is this related to the body class?

imanoleguskiza replied:
The issue with the patches across components will be solved in a follow up ticket, so for now I'm ok with merging this PR.

Medi4tor commented on tests/features/authentication.feature:
```diff
@@ -0,0 +1,10 @@
+@api
+Feature: Authentication
+  In order to allow personalized actions to be taken
```
I guess that was just a missing test... nothing related with body classes? You confirm?
